cumulusci/core/tests/BarTestPage.py

```diff
@@ -0,0 +1,18 @@
+"""
+This class is used by test_pageobjects
+"""
+from cumulusci.robotframework.pageobjects import BasePage
+from cumulusci.robotframework.pageobjects import pageobject
+
+
+@pageobject("Test", "Bar")
+class BarTestPage(BasePage):
+    object_name = "Bar"
```

Comment:
From the implementation of the decorator, it looks like it's not required to specify `object_name` here since it will be added from the decorator argument. I like that it's readable but not the duplication. How about recommending using the decorator like this:

    @pageobject(page_type="Test", object_name="Bar")

and not explicitly adding the attribute.

Reply:
I think that's probably a good suggestion. I'm on the fence about explicitly using the argument names, but you're definitely right about not setting `object_name`. I left the ability to explicitly set it mainly because I wanted to create some test classes where the name used to reference the page and the actual class were different. Not sure if that's ever going to be a valid use case.